routes/teams.js

```diff
@@ -34,10 +34,107 @@ router.get('/', function(res, res) {
 router.get('/team', function(req, res) {
     var term = req.query.q;
     console.log(term);
-    //find the team, This syntax does a sql-type like clause with case insensitivy(sp???) with the RegEx
+    //find the team, This syntax does a sql-type like clause with case insensitivity with the RegEx
     db.collection('teams').find({$or: [{'name': new RegExp(term, 'i')}, {'market': new RegExp(term, 'i')}]}).toArray(function (err, items) {
     	res.json(items);
     });
 });
 
+
+// when players endpoint is hit, call the API/DB using that team_id
+router.get('/players', function(req, res) {
+  team_id = req.query["team_id"];
+  players = fetchPlayers(team_id, res, returnPlayers);
+});
+
+var returnPlayers = function (players, res){
+    res.json(players);
+}
+
+  
+var fetchPlayers = function(team_id, res, callback){
+  db.collection('players').find({team_id : team_id}).toArray(function (err, items){
+    if (items.length > 0){ // data in Mongo
+      var date = items[0]["last_updated"].valueOf();
```

Comment:
no thats good. u just need to merge master into your branch. Your branch is so far behind that it doesnt have alot of the latest stuff. u will have lots of merge conflicts to resolve tho.